lls/cache.c

```diff
@@ -406,6 +506,14 @@ lls_req(enum lls_req_ty ty, void *req_arg)
 			sizeof(nd_req->pkts[0]) * nd_req->num_pkts);
 		break;
 	}
+	case LLS_REQ_PING6: {
+		struct lls_ping6_req *ping6_req =
+			(struct lls_ping6_req *)req_arg;
+		req->u.ping6 = *ping6_req;
+		rte_memcpy(req->u.ping6.pkts, ping6_req->pkts,
+			sizeof(ping6_req->pkts[0]) * ping6_req->num_pkts);
```

Comment:
Update the expression of `ele_size` in `lls_stage1()` to reflect this `rte_memcpy()`.